cernopendata_client/cli.py

```diff
@@ -68,6 +68,60 @@ def version():
     display_message(msg=__version__)
 
 
+def filter_metadata(output_field, filters, output_json):
+    # noqa: D301
+    """Filter metadata objects based on specified criteria.
+
+    :param field: Name of the array containing objects to access
```

Comment:
If the first argument is named `output_field`, then we should also update the docstring of the function, see the lines:

```
:param field:

...

:type field:
```